engine/gamesys/src/gamesys/components/comp_label.cpp

```diff
@@ -591,6 +601,21 @@ namespace dmGameSystem
         {
             return GetResourceProperty(dmGameObject::GetFactory(params.m_Instance), GetFontMap(component, component->m_Resource), out_value);
         }
+        else if (get_property == LABEL_PROP_LEADING)
+        {
+            out_value.m_Variant = dmGameObject::PropertyVar(component->m_Leading);
+            return dmGameObject::PROPERTY_RESULT_OK;
+        }
+        else if (get_property == LABEL_PROP_TRACKING)
+        {
+            out_value.m_Variant = dmGameObject::PropertyVar(component->m_Tracking);
+            return dmGameObject::PROPERTY_RESULT_OK;
+        }
+        else if (get_property == LABEL_PROP_LINE_BREAK)
+        {
+            out_value.m_Variant = dmGameObject::PropertyVar(component->m_LineBreak == 1);
```

Comment:
Coding style: In general, we consider ` != 0` to be true.

Reply:
Totally makes sense!